cocos/tween/actions/action-interval.ts

```diff
@@ -80,6 +82,11 @@ export abstract class ActionInterval extends FiniteTimeAction {
         }
     }
 
+    setStartTime (time: number): void {
```

Comment:
Does it support negative values?

Reply:
I think it will not, I will add a valid time checking in this function.

Reply:
If so, then i think don't have to add a variable to check if it is set or not, just check the value. The default value can be set to a negative value.

Reply:
Updated and added two unit tests for this.

Reply:
> If so, then i think don't have to add a variable to check if it is set or not, just check the value. The default value can be set to a negative value.

Good point. I think the default value could be 0, I will update the code.